reps/2023-02-03-node-affinity-feature-enhancements.md

```diff
@@ -0,0 +1,437 @@
+## Summary
+### General Motivation
+
+The current node affinity feature is relatively simple, it can only affinity with one node. 
```

Comment:
"it can only affinity" -> "it can only support affinity"

Reply:
done